backend/src/test/java/com/glebabramov/backend/service/ResumeServiceTest.java

```diff
@@ -203,6 +206,118 @@ void createResume_shouldReturn201Created_andTheCreatedResume_ifUserIsAdmin() {
 
 	}
 
+	@Nested
+	@DisplayName("updateResume()")
+	class updateResume {
+
+		@Test
+		@DirtiesContext
```

Comment:
You don't need @DirtiesContext in your unit tests. You are mocking all the things here so nothing actually changes. Applies to all test methods